### ISSUE TITLE
[tool_crash] _TypeError: (#0      ChromeProxyService._setUpChromeConsoleListeners.<anonymous closure> (package:dwds/src/services/chrome_proxy_service.dart:1351:49))

### ISSUE THREAD

**Issue by aleliottasyn** [flutter/flutter#134204]
## Command
```
flutter run -d chrome --web-port 8083
```

## Steps to Reproduce
1. ...
2. ...
3. ...
## exception

_TypeError: type 'Null' is not a subtype of type 'String' in type cast

## Logs
_TypeError: (#0      ChromeProxyService._setUpChromeConsoleListeners.<anonymous closure> (package:dwds/src/services/chrome_proxy_service.dart:1351:49))
```
#0      ChromeProxyService._setUpChromeConsoleListeners.<anonymous closure> (package:dwds/src/services/chrome_proxy_service.dart:1351:49)
#1      _rootRunUnary (dart:async/zone.dart:1407:47)
#2      _CustomZone.runUnary (dart:async/zone.dart:1308:19)
#3      _CustomZone.runUnaryGuarded (dart:async/zone.dart:1217:7)
#4      CastStreamSubscription._onData (dart:_internal/async_cast.dart:85:11)
#5      _rootRunUnary (dart:async/zone.dart:1407:47)
#6      _CustomZone.runUnary (dart:async/zone.dart:1308:19)
#7      _CustomZone.runUnaryGuarded (dart:async/zone.dart:1217:7)
#8      _BufferingStreamSubscription._sendData (dart:async/stream_impl.dart:339:11)
#9      _BufferingStreamSubscription._add (dart:async/stream_impl.dart:271:7)
#10     _SinkTransformerStreamSubscription._add (dart:async/stream_transformers.dart:63:11)
#11     _EventSinkWrapper.add (dart:async/stream_transformers.dart:13:11)
#12     WipDomain.eventStream.<anonymous closure>.<anonymous closure> (package:webkit_inspection_protocol/webkit_inspection_protocol.dart:377:22)
#13     _HandlerEventSink.add (dart:async/stream_transformers.dart:209:17)
#14     _SinkTransformerStreamSubscription._handleData (dart:async/stream_transformers.dart:111:24)
#15     _rootRunUnary (dart:async/zone.dart:1407:47)
#16     _CustomZone.runUnary (dart:async/zone.dart:1308:19)
#17     _CustomZone.runUnaryGuarded (dart:async/zone.dart:1217:7)
#18     _BufferingStreamSubscription._sendData (dart:async/stream_impl.dart:339:11)
#19     _DelayedData.perform (dart:async/stream_impl.dart:515:14)
#20     _PendingEvents.handleNext (dart:async/stream_impl.dart:620:11)
#21     _PendingEvents.schedule.<anonymous closure> (dart:async/stream_impl.dart:591:7)
#22     _rootRun (dart:async/zone.dart:1391:47)
#23     _CustomZone.run (dart:async/zone.dart:1301:19)
#24     _CustomZone.runGuarded (dart:async/zone.dart:1209:7)
```
```
[32m[✓][39m Flutter (Channel stable, 3.13.2, on macOS 12.2.1 21D62 darwin-x64, locale it-IT)
    [32m•[39m Flutter version 3.13.2 on channel stable at /Users/alessioliotta/flutter
    [32m•[39m Upstream repository https://github.com/flutter/flutter.git
    [32m•[39m Framework revision ff5b5b5fa6 (2 weeks ago), 2023-08-24 08:12:28 -0500
    [32m•[39m Engine revision b20183e040
    [32m•[39m Dart version 3.1.0
    [32m•[39m DevTools version 2.25.0

[33m[!][39m Android toolchain - develop for Android devices (Android SDK version 32.1.0-rc1)
    [32m•[39m Android SDK at /Users/alessioliotta/Library/Android/sdk
    [31m✗[39m cmdline-tools component is missing
      Run `path/to/sdkmanager --install "cmdline-tools;latest"`
      See https://developer.android.com/studio/command-line for more details.
    [31m✗[39m Android license status unknown.
      Run `flutter doctor --android-licenses` to accept the SDK licenses.
      See https://flutter.dev/docs/get-started/install/macos#android-setup for more details.

[31m[✗][39m Xcode - develop for iOS and macOS
    [31m✗[39m Xcode installation is incomplete; a full installation is necessary for iOS and macOS development.
      Download at: https://developer.apple.com/xcode/download/
      Or install Xcode via the App Store.
      Once installed, run:
        sudo xcode-select --switch /Applications/Xcode.app/Contents/Developer
        sudo xcodebuild -runFirstLaunch
    [31m✗[39m CocoaPods not installed.
        CocoaPods is used to retrieve the iOS and macOS platform side's plugin code that responds to your plugin usage on the Dart side.
        Without CocoaPods, plugins will not work on iOS or macOS.
        For more info, see https://flutter.dev/platform-plugins
      To install see https://guides.cocoapods.org/using/getting-started.html#installation for instructions.

[32m[✓][39m Chrome - develop for the web
    [32m•[39m Chrome at /Applications/Google Chrome.app/Contents/MacOS/Google Chrome

[32m[✓][39m Android Studio (version 2021.3)
    [32m•[39m Android Studio at /Applications/Android Studio.app/Contents
    [32m•[39m Flutter plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/9212-flutter
    [32m•[39m Dart plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/6351-dart
    [32m•[39m Java version OpenJDK Runtime Environment (build 11.0.13+0-b1751.21-8125866)

[32m[✓][39m VS Code (version 1.81.1)
    [32m•[39m VS Code at /Applications/Visual Studio Code.app/Contents
    [32m•[39m Flutter extension version 3.72.0

[32m[✓][39m Connected device (3 available)
    [32m•[39m SM A137F (mobile) • R58TB0S9KAA • android-arm    • Android 12 (API 31)
    [32m•[39m macOS (desktop)   • macos       • darwin-x64     • macOS 12.2.1 21D62 darwin-x64
    [32m•[39m Chrome (web)      • chrome      • web-javascript • Google Chrome 116.0.5845.140

[32m[✓][39m Network resources
    [32m•[39m All expected network resources are available.

[33m![39m Doctor found issues in 2 categories.

```

## Flutter Application Metadata
**Type**: malformed
**Version**: 1.0.6+1
**Material**: true
**Android X**: false
**Module**: false
**Plugin**: false
**Android package**: null
**iOS bundle identifier**: null
**Creation channel**: null
**Creation framework version**: null
### Plugins
flutter_secure_storage-9.0.0
flutter_secure_storage_linux-1.2.0
flutter_secure_storage_macos-3.0.1
flutter_secure_storage_web-1.1.2
flutter_secure_storage_windows-3.0.0
package_info_plus-4.1.0
path_provider-2.1.1
path_provider_android-2.2.0
path_provider_foundation-2.3.1
path_provider_linux-2.2.1
path_provider_windows-2.2.1

**Comment by dam-ease** [comment]
Hi @aleliottasyn. I see there's an open issue in the dart-lang/webdev repository that is addressing the case you described https://github.com/dart-lang/webdev/issues/2161.

Closing this issue in favour of that one, so please follow up on it for further updates. Thanks!

**Comment by aleliottasyn** [comment]
Hi @dam-ease I see that the bug is fixed in 19.0.2, but flutter keeps downloading the buggy version(19.0.1+1). So I'm not able to update my app. Please let me figure out how to fix this.

**Comment by dam-ease** [comment]
Thanks for your response @aleliottasyn. 

>  but flutter keeps downloading the buggy version(19.0.1+1)

Not sure as to why this occurs, you might want to file another issue if this persists.
